recipes/cpp-sort/all/conanfile.py

```diff
@@ -80,4 +87,4 @@ def package_info(self):
             self.cpp_info.cxxflags = ["/permissive-"]
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()  # Header-only
```

Comment:
```suggestion
        self.info.clear()
```
It's well documented already.

Reply:
Fair